### PR TITLE
Rename parallel downloads flags

### PR DESCRIPTION
**Pull request by sethiay**
### Description
Rename parallel downloads flags to:
```
enable-parallel-downloads
parallel-downloads-per-file
max-parallel-downloads
download-chunk-size-mb
enable-crc
```

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - NA
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

File: internal/cache/file/downloader/jm_parallel_downloads_test.go [hunk 1]
```diff
@@ -92,37 +92,37 @@ func createObjectInStoreAndInitCache(t *testing.T, cache *lru.Cache, bucket gcs.
...
+			name:                     "download in chunks of concurrency * readReqSize",
+			objectSize:               15 * util.MiB,
+			readReqSize:              3,
+			parallelDownloadsPerFile: math.MaxInt,
+			maxParallelDownloads:     3,
+			subscribedOffset:         7,
+			downloadOffset:           10,
+			// Concurrency can go to (maxParallelDownloads + 1) in case
+			// parallelDownloadsPerFile > maxParallelDownloads because we always
 			// spawn minimum of 1 go routine per async job.
```
Comment (kislaykishore) [inline internal/cache/file/downloader/jm_parallel_downloads_test.go:114]: Nit
```suggestion
			// spawn a minimum of 1 go routine per async job.
```

File: internal/cache/file/downloader/parallel_downloads_job_test.go [hunk 2]
```diff
@@ -47,7 +47,7 @@ func init() { RegisterTestSuite(&parallelDownloaderTest{}) }
 
 func (dt *parallelDownloaderTest) SetUp(*TestInfo) {
 	dt.defaultFileCacheConfig = &config.FileCacheConfig{EnableParallelDownloads: true,
-		DownloadParallelismPerFile: 3, ReadRequestSizeMB: 3, EnableCrcCheck: true}
+		ParallelDownloadsPerFile: 5, DownloadChunkSizeMB: 2, EnableCRC: true}
```
Comment (kislaykishore) [inline internal/cache/file/downloader/parallel_downloads_job_test.go:50]: Why do we need to change these values?

**Review by kislaykishore — CHANGES_REQUESTED** [review]

File: internal/cache/file/downloader/parallel_downloads_job_test.go (hunk 2 above)
Comment (sethiay) [reply internal/cache/file/downloader/parallel_downloads_job_test.go]: We don't need to change these values, I have put them back.

**Review by kislaykishore — APPROVED** [review]

**Review by ashmeenkaur — APPROVED** [review]
Relying on @kislaykishore's review

**Pull request merged**